src/main/java/com/exasol/releaserobot/usecases/UserInput.java

```diff
@@ -191,4 +191,8 @@ private void throwExceptionForMissingParameter(final String exceptionCode, final
             throw new IllegalArgumentException(message);
         }
     }
+
+    public String getRepositoryFullName() {
+        return this.getRepositoryOwner() + "/" + this.repositoryName;
```

Comment:
Given that this is the only use of `RepositoryOwner`, I wonder if it should exists at all. Maybe we can remove it, and deal only with `repositoryName`, and users should specify the full repository name in the command line.

Reply:
the owner is hardcoded now (exasol), and the user doesn't  input it at all for convenience

Reply:
We agreed to do it in another PR